partcad/tests/unit/test_git_clone_retries.py

```
@@ -113,8 +121,15 @@ def side_effect(*args, **kwargs):
         return MagicMock()
     side_effect.counter = 0
 
+    def git_config(*args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for private function `git_config`

(ANN202)